src/main/java/org/alliancegenome/curation_api/dao/ontology/NcbiTaxonTermDAO.java

```diff
@@ -29,9 +30,21 @@ public NCBITaxonTerm downloadAndSave(String taxonCurie) {
             return null;
         }
         
-        NCBITaxonResponseDTO resp = api.getTaxonFromNCBI("taxonomy", "json", taxonIdMatcher.group(1));
-        HashMap<String, Object> result = resp.getResult();
-        HashMap<String, Object> taxonMap = (HashMap<String, Object>)result.get(taxonIdMatcher.group(1));
+        HashMap<String, Object> taxonMap = null;
+        int attempts = 5;
```

Comment:
Could you name it something lik MAX_ATTEMPTS and make it const ?

Reply:
Done